conf/server.yaml.template

```diff
@@ -8,6 +8,8 @@ SERVER:
     # RELEASE: populate with satellite version
     # The snap version currently testing (if applicable)
     # SNAP:
+    # The RHEL Base OS Version(x.y) where the Satellite is installed
+    # RHEL_VERSION: "7.9"
```

Comment:
Leave the default value as blank similar to satellite release and snap versions!